examples/experimental/bert_squad_pytorch/const.yaml

```diff
@@ -0,0 +1,27 @@
+description: Bert_SQuAD_PyTorch
+hyperparameters:
+    global_batch_size: 24
+    learning_rate: 3e-5
+    lr_scheduler_epoch_freq: 1
+    model_type: 'bert'
+    adam_epsilon: 1e-8
+    weight_decay: 0
+    num_warmup_steps: 0
+    num_training_steps: 400
+    max_seq_length: 384
+    doc_stride: 128
+    max_query_length: 64
+    n_best_size: 20
+    max_answer_length: 30
+    null_score_diff_threshold: 0.0
+searcher:
+    name: single
+    metric: f1
+    max_steps: 2
```

Comment:
blocking: let's set a more realistic amount of steps that converges to a reasonable f1 (whatever that is on this problem). add a comment at the top of the config calling out what metrics a user can expect.